pkg/models/repository_configuration.go

```diff
@@ -1,13 +1,15 @@
 package models
 
+import "github.com/lib/pq"
+
 type RepositoryConfiguration struct {
 	Base
-	Name      string `json:"name" gorm:"default:null"`
-	URL       string `json:"url" gorm:"default:null"`
-	Version   string `json:"version" gorm:"default:null"`
-	Arch      string `json:"arch" gorm:"default:null"`
-	AccountID string `json:"account_id" gorm:"default:null"`
-	OrgID     string `json:"org_id" gorm:"default:null"`
+	Name      string         `json:"name" gorm:"default:null"`
+	URL       string         `json:"url" gorm:"default:null"`
+	Versions  pq.StringArray `json:"version" gorm:"type:text[],default:null"`
```

Comment:
what's the difference between pq.StringArray and normal go array?

Reply:
for Gorm, you can't use a normal go string array, it doesn't know how to map it to postgres's varchar array.  This special type implements some methods that gorm needs to translate it properly (https://gorm.io/docs/data_types.html) .  That said, its just a string array under the hood:  https://github.com/lib/pq/blob/master/array.go#L626   and as you'll see in the other code, you can assign a string array to it.

Reply:
thanks!